### PR TITLE
Toast & rating component

### PR DESCRIPTION
**Pull request by rushikesh3299**
Toast Component
https://deploy-preview-8--saraniui.netlify.app/components/toast/toast

Rating component
https://deploy-preview-8--saraniui.netlify.app/components/rating/rating

### CODE REVIEW

File: Components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import url("/Common_CSS/base.css");
+
+.toast{
+    padding:1rem;
+    border-radius:10px;
+    background: var(--color-warning);
+    color:#fff;
```
Comment (sushil-mod) [inline Components/Toast/toast.css]: if possible use color variable

File: Components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    align-items:center;
+  }
+  
+  .toast a{
+    text-decoration:none;
+    color:var(--color-primary);
+  }
+  
+  .toast button{
+    background:#333;
```
Comment (sushil-mod) [inline Components/Toast/toast.css]: same here too..

**Review by sushil-mod — APPROVED** [review]
added some comment ,then good to go merge!

File: Components/Toast/toast.css (hunk 1 above)
Comment (rushikesh3299) [reply Components/Toast/toast.css]: Done

File: Components/Toast/toast.css (hunk 2 above)
Comment (rushikesh3299) [reply Components/Toast/toast.css]: Done

**Review by Kothalkarpriya — APPROVED** [review]
Looks good to me

**Pull request merged**